Weather Alert.xcodeproj/project.pbxproj.orig

```diff
@@ -0,0 +1,553 @@
+// !$*UTF8*$!
+{
+	archiveVersion = 1;
+	classes = {
+	};
+	objectVersion = 46;
+	objects = {
+
+/* Begin PBXBuildFile section */
+<<<<<<< HEAD
```

Comment:
Will try helping you out if I'll have time today, or please ask @OreniOS for help